base/namedtuple.jl

```diff
@@ -350,6 +350,9 @@ julia> @NamedTuple begin
        end
 NamedTuple{(:a, :b),Tuple{Float64,String}}
 ```
+
+!!! compat "Julia 1.5"                                                          
```

Comment:
Trailing whitespace.

Reply:
whoops, downside of the github in-browser editor